config.json

```diff
@@ -228,8 +228,8 @@
         "status": "beta"
       },
       {
```

Comment:
I only changed the `name` and `slug` for now. Perhaps I should also change the `prerequisites` as well?